primitives/blockchain/src/backend.rs

```diff
@@ -91,7 +91,7 @@ pub trait Backend<Block: BlockT>:
 	/// Get block body. Returns `None` if block is not found.
 	fn body(&self, hash: &Block::Hash) -> Result<Option<Vec<<Block as BlockT>::Extrinsic>>>;
 	/// Get block justifications. Returns `None` if no justification exists.
-	fn justifications(&self, id: BlockId<Block>) -> Result<Option<Justifications>>;
+	fn justifications(&self, hash: &Block::Hash) -> Result<Option<Justifications>>;
```

Comment:
```suggestion
	fn justifications(&self, hash: Block::Hash) -> Result<Option<Justifications>>;
```

`Hash` implements `Copy`, so we can take by value.